code/modules/surgery/organs/augments_chest.dm

```diff
@@ -5,43 +5,6 @@
 	implant_overlay = "chest_implant_overlay"
 	zone = "chest"
 
-/obj/item/organ/cyberimp/chest/nutriment
```

Comment:
dont worry if this gets merged ill make an rnd implant that just shoves soylent into your metabolism for the remainder of the round